CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Cromwell Change Log
 
+## 55 Release Notes
+
+### Bug Fixes
+
+* Fixed a bug where `read_json()` failed for `Array[_]` inputs. It should now work for all valid JSON data inputs. 
```

Comment:
I feel like we revised the 54 release notes in the previous PR, maybe it got lost in git? If you chose not to implement the suggestion I'm not offended either 😄

Reply:
Oh that is weird! I do remember committing your suggestion!

Reply:
Oh I think I know what happened! So I committed your suggestion through Github, but then I had to make some other changes and when I rebased it against develop I forced pushed all changes without pulling the commit with your suggestion onto local branch. And hence that commit never made it to develop :(

I will update this CHANGELOG with your suggestion from previous PR.